aura/doc/aura.conf

```diff
@@ -34,6 +34,8 @@
 # |------+------------|
 # | eo   | Esperanto  |
 # |------+------------|
+# | uk   | Ukrainian  |
```

Comment:
Why is it in a separate section? I think it cat fit in nicely in the Croatian..Serbian section.

Reply:
I'll look this over locally.